app/services/dttp/batch_create.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Dttp
+  class BatchCreate
+    class << self
```

Comment:
We should extract this to a `ServicePattern` module as we do in TTAPI & Publish (https://github.com/DFE-Digital/teacher-training-api/blob/master/app/services/service_pattern.rb)